src/render_tasks/d3d12_rtao_task.hpp

```diff
@@ -16,6 +16,13 @@
 
 namespace wr
 {
+	namespace rtaoSettings
```

Comment:
style guide.

Reply:
Which style? Style guide is inconsistent and uses both snake and pascal case

Reply:
Namespace is removed entirely